nodes/McpClient/McpClient.node.test.ts

```
@@ -0,0 +1,256 @@
+	// console.log(`DESCRIBE: ${name}`);
+	fn();
+}
+function it(name: string, fn: () => void) {
+	try {
+		// console.log(`  IT: ${name}`);
+		fn();
+		// console.log(`  PASS: ${name}`);
+	} catch (e: any) {
+		// console.error(`  FAIL: ${name}`);
+		// console.error(e.message);
+		throw e; // Re-throw to make it clear a test failed if running in a context that cares
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)